delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -0,0 +1,465 @@
+            err_arr = self.handle_detail(error_infos, 'events')
+            if err_arr:
+                for alert_map in err_arr:
+                    time_stamp = alert_map.get('time-stamp-numeric')
+                    occur_time = int(time_stamp) * self.SECONDS_TO_MS
+                    if not alert_util.is_alert_in_time_range(query_para,
+                                                             occur_time):
+                        continue
+                    alert_name = alert_map.get('message', '')
+                    event_id = alert_map.get('event-id')
+                    location = alert_map.get('serial-number', '')
+                    resource_type = alert_map.get('event-code', '')
+                    severity = alert_map.get('severity')
+                    if severity == 'Informational' or severity is None:
+                        continue
+                    alert_model = {
+                        'alert_id': event_id,
+                        'alert_name': alert_name,
+                        'severity': severity,
+                        'category': constants.Category.FAULT,
+                        'type': 'EquipmentAlarm',
+                        'sequence_number': alert_map.get('event-code'),
+                        'occur_time': occur_time,
+                        'description': alert_name,
+                        'resource_type': resource_type,
+                        'location': location
+                    }
+                    alert_list.append(alert_model)
+            return self.repeated_err(alert_list)
+        except Exception as e:
+            err_msg = "Failed to get storage error: %s" % (six.text_type(e))
+            LOG.error(err_msg)
+            raise e
+
+    def repeated_err(self, detail_info):
+        exist_questions = set()
+        detail_arr = []
+        for item in detail_info:
+            question = item['resource_type']
+            if question not in exist_questions:
+                exist_questions.add(question)
+                detail_arr.append(item)
+        return detail_arr
+
+    @staticmethod
+    def parse_alert(alert):
+        try:
+            alert_model = dict()
+            alert_id = ''
+            description = ''
+            severity = SSHHandler.TRAP_SEVERITY_MAP.get('8')
+            seq_num = ''
+            event_type = ''
+            for alert_key, alert_value in alert.items():
+                if SSHHandler.OID_ERR_ID in alert_key:
+                    alert_id = str(alert_value)
+                elif SSHHandler.OID_EVENT_TYPE in alert_key:
+                    event_type = alert_value
+                elif SSHHandler.OID_EVENT_DESC in alert_key:
+                    description = alert_value
+                elif SSHHandler.OID_SEVERITY in alert_key:
+                    severity = SSHHandler.TRAP_SEVERITY_MAP\
+                        .get(alert.get(SSHHandler.OID_SEVERITY),
+                             constants.Severity.INFORMATIONA)
```

Comment:
There is no `INFORMATIONA` in class `constants.Severity` .

Reply:
Change to an existing value